ferret/explainers/explanation_speech/gradient_speech_explainer.py

```diff
@@ -181,7 +177,7 @@ def compute_explanation(
             scores=scores,
             explainer=self.NAME + "-" + aggregation,
             target=targets if n_labels > 1 else targets,
-            audio_path=audio_path,
+            audio_path=audio,
```

Comment:
Let's change the name of this optional argument in to `audio`, for consistency. This needs to be done in the `ExplanationSpeech` class within `explanation_speech.py`.